configs/milvus.yaml

```diff
@@ -329,6 +329,10 @@ dataNode:
     # The period to sync segments if buffer is not empty.
     syncPeriod: 600 # Seconds, 10min
 
+  memory:
+    forceSyncEnable: true # `true` to force sync if memory usage is too high
+    forceSyncThreshold: 0.7 # forceSync only take effects when memory usage ratio > forceSyncThreshold  
```

Comment:
for cluster deployment we could set it to 0.7, the other 30% is left for bulkload and compaction.

For standalone deployment we should set it to 0.2 because querynode will have one copy of growing segment and datanode will has another copy.